openshift/release/update-to-head.sh

```diff
@@ -64,7 +64,7 @@ git push -f ${OPENSHIFT_REMOTE} release-next-ci
 if hash hub 2>/dev/null; then
    # Test if there is already a sync PR in 
    COUNT=$(hub api -H "Accept: application/vnd.github.v3+json" repos/openshift/${REPO_NAME}/pulls --flat \
-    | grep -c ":robot: Triggering CI on branch 'release-next' after synching to upstream/main") || true
+    | grep -c ":robot: Triggering CI on branch 'release-next' after synching to upstream/ma") || true
```

Comment:
🤔

Reply:
Hey @vdemeester just making the search find either the `master` or `main` branch in the PR summary

Reply:
@vdemeester how do you like it now?

Reply:
we could have `grep -E '(master|main)'` too for explicitness (just a nit)